charts/k8ssandra/values.yaml

```diff
@@ -13,10 +13,11 @@ cassandra:
   # management-api images. If you do want to change one of these mappings, the new value
   # should be a management-api image.
   versionImageMap:
-    3.11.7: datastax/cassandra-mgmtapi-3_11_7:v0.1.19
-    3.11.8: datastax/cassandra-mgmtapi-3_11_8:v0.1.19
-    3.11.9: datastax/cassandra-mgmtapi-3_11_9:v0.1.19
-    3.11.10: datastax/cassandra-mgmtapi-3_11_10:v0.1.19
+    3.11.7: datastax/cassandra-mgmtapi-3_11_7:v0.1.22
+    3.11.8: datastax/cassandra-mgmtapi-3_11_8:v0.1.22
+    3.11.9: datastax/cassandra-mgmtapi-3_11_9:v0.1.22
+    3.11.10: datastax/cassandra-mgmtapi-3_11_10:v0.1.22
+    4.0.0: datastax/cassandra-mgmtapi-4_0_0:v0.1.22
```

Comment:
nit: Maybe add a comment on line 7 that 4.0.0 is also supported.
https://github.com/k8ssandra/k8ssandra/pull/336/files#diff-bb025298fadbc3d3f32ec53fad275576fd87b5ed47e1aa66f6363c56a488df7eR7

Reply:
👍